package.json

```diff
@@ -122,7 +122,7 @@
     "styled-components": "^5.0.1",
```

Comment:
Can't comment on unchanged lines in GitHub's code review, but the line above this, `rollup-plugin-postcss`, should also be updated to a version beyond [v3](https://github.com/egoist/rollup-plugin-postcss/releases/tag/v3.0.0) as that supports Rollup 2 (and not Rollup 1).

That would also resolve the final PR needed for Rollup 2, #693 (also mentioned in https://github.com/formium/tsdx/issues/821#issue-684122764).

Reply:
Looks like this was added in the most recent update to the PR